app/models/miq_worker/systemd_common.rb

```diff
@@ -12,8 +12,8 @@ def supports_systemd?
       end
 
       def ensure_systemd_files
-        target_file_path.write(target_file) unless target_file_path.exist?
-        service_file_path.write(unit_file) unless service_file_path.exist?
+        target_file_path.write(target_file)
+        service_file_path.write(unit_file)
```

Comment:
Were these `.exist?` checks not needed previously?

Reply:
Answered here https://github.com/ManageIQ/manageiq/pull/20824#discussion_r526945100